src/stores/ConfigStore.ts

```diff
@@ -20,6 +20,7 @@ type CleengData = {
 type ConfigState = {
   config: Config;
   accessModel: AccessModel;
+  environments: { host: string; player: string } | null;
```

Comment:
Naming suggestion to make singular and don't allow it to be null (initial value can be the production defaults)
```suggestion
  environment: { host: string; player: string };
```

Reply:
@dbudzins env values are optional (not present for all modes) so I chose a local constant to avoid `undefined`.